src/documentBuilder.ts

```diff
@@ -2,7 +2,11 @@ import dayjs = require('dayjs');
 import {createHash} from 'crypto';
 import {CompressionType, Document, Metadata, MetadataValue} from './document';
 import {SecurityIdentityBuilder} from './securityIdentityBuilder';
-
+import {isFieldNameValid} from './fieldAnalyser/fieldUtils';
+import {UnsupportedFieldError} from './errors/fieldErrors';
+import {BuiltInTransformers} from './validation/transformation/builtInTransformers';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
:suspect:

Reply:
For typedoc.
I need to import the type if I want to reference it in the documentation using links
![image](https://user-images.githubusercontent.com/12199712/168254746-4b9ac1ef-6a24-4376-8235-224e79e1b8f3.png)

Reply:
Yeah, ok. I tried some stuff but indeed that is a limitation with TS today.

To at least isolate this, do the import like this:
```ts
// Used for Documentation.
// eslint-disable-next-line @typescript-eslint/no-unused-vars
import type {BuiltInTransformers} from './validation/transformers/transformer';
```